package.json

```diff
@@ -7,7 +7,11 @@
     "url": "git+https://github.com/nearform/temporal_tables.git"
   },
   "scripts": {
-    "update-version": "node ./scripts/update-version.js"
+    "update-version": "node ./scripts/update-version.js",
+    "pretest": "docker run --name temporal-tables-test -e POSTGRES_PASSWORD=password -e POSTGRES_USER=postgres -p 5432:5432 -d postgres:latest && sleep 3 && docker exec temporal-tables-test pg_isready -h localhost -p 5432 -U postgres -t 30",
+    "test": "(PGHOST=localhost PGPORT=5432 PGUSER=postgres PGPASSWORD=password make run_test; ret=$?; docker stop temporal-tables-test && docker rm temporal-tables-test; exit $ret)",
+    "test:nochecks": "npm run pretest && (PGHOST=localhost PGPORT=5432 PGUSER=postgres PGPASSWORD=password make run_test_nochecks; ret=$?; docker stop temporal-tables-test && docker rm temporal-tables-test; exit $ret)",
+    "cleanup": "docker stop temporal-tables-test || true && docker rm temporal-tables-test || true"
```

Comment:
why do we need new npm scripts?

Reply:
Thanks. This script is not cross platform, wherease a docker compose file will be. The only thing we may want to mention in the dock is to spin it up for development

Reply:
I always forget about anything non mac 👍

Reply:
Feel free to merge once you're done with this change. It's a minor thing compared to the actual feature